### PR TITLE
Feature logic untag 

### PR DESCRIPTION
**Pull request by rmj1405**
PENDING: UntagFoodCommand execute_untagFood_success() test

### CODE REVIEW

**Comment by Jnjy** [comment]
@rmj1405 it seems that the code failed the checkstyle stage, can you review it again?

File: src/main/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommand.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    public UntagFoodCommand(String tagName, Index index) {
+        requireNonNull(tagName);
+        requireNonNull(index);
+        this.tag = new Tag(tagName);
+        this.index = index;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        Food foodToUntag = UntagFoodCommand.getFoodToUntag(model, tag, index);
```
Comment (Jnjy) [inline src/main/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommand.java:41]: Will need to have a `requireNonNull` check for the model

File: src/test/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+import seedu.wife.logic.commands.generalcommands.ExitCommand;
+import seedu.wife.model.Model;
+import seedu.wife.model.ModelManager;
+import seedu.wife.model.UserPrefs;
+import seedu.wife.model.tag.Tag;
+import seedu.wife.testutil.TagBuilder;
+import seedu.wife.testutil.TypicalTag;
+
+
```
Comment (Jnjy) [inline src/test/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommandTest.java:21]: Can you remove the extra new line here?

**Review by Jnjy — CHANGES_REQUESTED** [review]
Just minor changes required, everything else looks good!

File: src/test/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommandTest.java (hunk 2 above)
Comment (rmj1405) [reply src/test/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommandTest.java]: ✅

File: src/main/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommand.java (hunk 1 above)
Comment (rmj1405) [reply src/main/java/seedu/wife/logic/commands/tagcommands/UntagFoodCommand.java:42]: done!

**Review by Jnjy — APPROVED** [review]
LGTM

**Pull request merged**